### ISSUE TITLE
Heading 2 Font Changes

### ISSUE THREAD

**Issue by Rodlund** [colineckert/obsidian-things#63]
Hello!
I'm not a CSS wizard. After looking through the CSS I couldn't find or easily identify the section that changes H2 font. I will post an image but I would like to edit out the font being changed for Heading 2 and don't know how to easily modify it. In the image below it's the 2nd line.

![image](https://user-images.githubusercontent.com/106779482/171861160-e0c61d17-617c-446c-87c8-61c6d004a61d.png)

**Comment by colineckert** [comment]
Hey @Rodlund happy to help here. What exactly do you want to change about the theme's treatment for H2's? For example, do you not want there to be an underline for H2? Lmk and I will try and point you to where you'll need to make changes.

**Comment by colineckert** [comment]
Nope, it's the same font as the other headings, but it does have an underline and font-weight of 700 (bold) like the H1. If you want to adjust that, the simplest way would be to install the Style Settings plugin and change the font-weight value to your desired weight.
<img width="745" alt="Screen Shot 2022-06-12 at 5 25 00 PM" src="https://user-images.githubusercontent.com/60206665/173259901-82346f99-5681-41f1-a723-f5f115749a68.png">

If it's the underline you want to remove, you can just comment out these lines:
- https://github.com/colineckert/obsidian-things/blob/main/obsidian.css#L472
- https://github.com/colineckert/obsidian-things/blob/main/obsidian.css#L542

**Comment by Rodlund** [comment]
Ah I see what you are saying now. It may have just been my screen with the
color but the font looked different from my end. You can delete this issue
:)
